zapier/authentication.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  type: 'custom',
+  test: {
+    removeMissingValuesFrom: { body: false, params: false },
+    url: '{{process.env.BASE_URL}}/api/zapier/validate',
+  },
+  fields: [
+    {
```

Comment:
![Enter your API key, located at akismet comaccount](https://github.com/user-attachments/assets/3a7721d1-eb52-4c29-9f8b-9a070c07b321)

Can we add a help text with a link to [opnform.com/settings/access-tokens](http://opnform.com/settings/access-tokens)